ddtrace/tracer/option.go

```diff
@@ -537,12 +541,21 @@ func WithDebugMode(enabled bool) StartOption {
 }
 
 // WithLambdaMode enables lambda mode on the tracer, for use with AWS Lambda.
+// This option is only required if the the Datadog Lambda Extension is not
+// running.
 func WithLambdaMode(enabled bool) StartOption {
 	return func(c *config) {
 		c.logToStdout = enabled
 	}
 }
 
+// WithSendRetries enables
```

Comment:
Maybe something like
```
// WithSendRetries enables re-sending payloads that are not successfully submitted to the agent.
// This will cause the tracer to retry the send at most `retries` times.
```

Just a suggestion.